lib/rubygems/commands/signout_command.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+require 'rubygems/command'
+require 'rubygems/commands/query_command'
+
+class Gem::Commands::SignoutCommand < Gem::Commands::QueryCommand
```

Comment:
good catch, @simi! I'm pretty sure you pointed that out earlier and I forgot to make sure it was fixed. 😓 

If nobody gets to it in the meantime, I'll make a PR fixing this either later today or tomorrow.